src/cocotb/handle.py

```diff
@@ -565,6 +565,13 @@ class _GPISetAction(enum.IntEnum):
     RELEASE = 2
 
 
+class _GPIResolveX(enum.IntEnum):
```

Comment:
This is private, but it is something that users are expected to use? I might instead use `DocEnum` to document this, while also inheriting from `str` to mimic `StrEnum`.

Reply:
I changed this to `DocEnum`, but did not inherit from `str` as the `int` value gets passed through to the GPI.  However, I'm not sure I'm using `DocEnum` correctly yet.  I've added this class to the docs, but the enum values aren't enumerated or described.

Also, I removed the `_` since this is user facing.

Reply:
Yeah, using `int` makes more sense here now that I have a better look at it.